DOCS/interface-changes/external-tracks.txt

```diff
@@ -0,0 +1,4 @@
+deprecate `--autoload-files`
+add `--sub-file-priority`
+add `--audio-file-priority`
+add `--video-file-priority`
```

Comment:
Personally not sure if we need three separate options, it seem like too much fine-grained control. External file is external file. But whatever if you like to have it like that.